File-Formats/XlsIO/Excel-to-PDF-Conversion.md

```diff
@@ -391,11 +391,11 @@ A complete working example to convert Excel with comments to PDF and render comm
 
 The following screenshot represents the input Excel file with notes
 
-![Input excel file](Excel-to-PDF-Conversion_images/Excel-to-PDF-Conversion_img2.png)
+![Input excel file in File Formats.](excel-to-pdf-conversion_images/file-formats-input-excel-file.png)
```

Comment:
Have you changed the folder name? check in all places

Reply:
Yes, folder name changes